app/assets/javascripts/darkswarm/all.js.coffee

```diff
@@ -9,6 +9,7 @@
 #= require angular-animate
 #= require angular-resource
 #= require lodash.underscore.js
+#= require bluebird.js
```

Comment:
A comment here stating this is required by angular-google-maps.min.js, would come in handy.

Reply:
Comment added :+1: